CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.0 (2021-05-31)
+
+- Add `Bambora::V1::PaymentResource#get` method
```

Comment:
Cutting a new release, `0.4.0`, to accommodate this additional feature. Once this is merged, I will tag `main` appropriately, triggering a new release to be pushed to RubyGems through the `gempush` GitHub action.